clippy_lints/src/casts/cast_possible_truncation.rs

```diff
@@ -79,6 +80,21 @@ fn apply_reductions(cx: &LateContext<'_>, nbits: u64, expr: &Expr<'_>, signed: b
                 nbits
             }
         },
+        ExprKind::Path(ref _qpath) => get_constant_bits(cx, expr).unwrap_or(nbits),
```

Comment:
`get_constant_bits` isn't constrained to the current crate since it ultimately calls `constant` and not `constant_simple`.

Reply:
`constant_simple` only works for expressions of literals, not const items, if I understand correctly.
I don't see why constant items should be constrainted to the current crate?

Reply:
A constant from another crate might not have the same value for all versions of the crate. This isn't a problem for constants in the current crate since a crate can't compile against multiple versions of itself.

Reply:
Okay, that makes sense.